src/libsyntax_ext/util.rs

```diff
@@ -0,0 +1,11 @@
+use syntax_pos::Symbol;
```

Comment:
I know this is a tiny module but it doesn't really fit anywhere else except maybe `lib.rs` but that file is pretty much only about "register all the things" which the function below is not about.